### ISSUE TITLE
What files should be ignored

### ISSUE THREAD

**Issue by porfirioribeiro** [greenbits/brick#29]
On the [documentation](https://github.com/greenbits/brick#recommended-but-optional) you say that every `*.g.dart` should be ignored, but in some files they say that it should not be ignored like [schema.g.dart](https://github.com/greenbits/brick/blob/master/packages/brick_offline_first/example/lib/app/db/schema.g.dart#L2) on the example.

Would be nice to have this clarified.

**Comment by tshedor** [comment]
@porfirioribeiro Thanks using Brick.

Every `*.g.dart` should be ignored, including `schema.g.dart`. I committed the generated files in the example to accessibly illustrate what Brick's outputs. You're right that this is confusing and seemingly counter to the documentation. I'll open a PR in a minute to note that. 

Many thanks for the catch, and good luck with using Brick. Please drop me a line at tshedor at greenbits dot com if you run into any other obstacles or have suggestions on an improved API.

**Comment by porfirioribeiro** [comment]
Thanks, so far I am enjoying it, was annoyed by codegen not working ok, but then I read somewhere that I needed to run build twice, and everything is going good.
I'm still just doing mostly read only resources and cache for offline, next week I hope to start implementing some resources that would need to be able to create offline, the reason I started using brick the first place! I'll contact you if I have any doubts.

Thanks!
